sentinelsat/sentinel.py

```diff
@@ -1354,6 +1453,13 @@ def _parse_odata_timestamp(in_date):
     return datetime.utcfromtimestamp(seconds) + timedelta(milliseconds=ms)
 
 
+def _get_quicklook_url(id):
+    url = "https://scihub.copernicus.eu/apihub/odata/v1/Products('{id}')/Products('Quicklook')/$value".format(
+        id=id
+    )
```

Comment:
This will fail if the user provided a different SciHub URL, e.g. https://apihub
.copernicus.eu

Please re-use the url attribute of the api class, which is where we handle the SciHub base-url logic.

Reply:
Thanks for catching that, updated to use self.api_url and added the quicklook feature to changelog as well